server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const cors = require('cors');
+const app = express();
+app.use(cors());
+
+app.use(express.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    res.send('Hello, World!');
+});
+
+app.listen(3001, () => {
```

Comment:
set 3001 as a constant on top, between line 3 & 4

something like 

```
const PORT = 3001
```

And then use it as a variable reference at line 13 on app.listen